packages/backend/src/handlers/actions/routes.ts

```diff
@@ -1,17 +1,14 @@
 import express from 'express';
 
 import { asyncHandlerWrapper } from '../../lib/apiHelpers';
+import { cacheRoutes } from './3Box/routes';
 import { guildRoutes } from './guild/routes';
 import { migrateSourceCredAccounts } from './migrateSourceCredAccounts/handler';
 import { questsRoutes } from './quests/routes';
-import { updateBoxProfileHandler } from './updateBoxProfile/handler';
 
 export const actionRoutes = express.Router();
 
-actionRoutes.post(
-  '/updateBoxProfile',
-  asyncHandlerWrapper(updateBoxProfileHandler),
-);
+actionRoutes.use('/cache', cacheRoutes);
```

Comment:
route should have a more descriptive name like `/cacheIdxProfiles`

Reply:
It is now `idxCache/`.